src/audio/detect_test.c

```diff
@@ -654,7 +545,7 @@ struct comp_driver comp_keyword = {
 	},
 };
 
-static void sys_comp_keyword_init(void)
+UT_STATIC void sys_comp_keyword_init(void)
```

Comment:
Looks like this + changes in src/include/sof/audio/detect_test.h are fix for making symbol accessible from tests. Can it be done in separate commit? First fix then refactor?